scripts/fast-reboot

```diff
@@ -271,14 +271,23 @@ function teardown_control_plane_assistant()
     fi
 }
 
+function is_secureboot() {
+    grep -Eq 'secure_boot_enable=[1y]' /proc/cmdline
```

Comment:
This implies the current image is boot up with secure boot.

What if a device is booting from none-secure-boot image into secure boot image? Does fast-reboot work there?

Reply:
You would need to go in Aboot and reconfigure the secureboot setting via the console.
It is therefore not possible to fast-reboot from non-secure to secure mode. Which is why it hasn't been handled here.